packages/typescript/client/src/main.ts

```diff
@@ -61,24 +88,64 @@ export class SessionBackend {
 
       const messages = value
         .split('\n')
-        .map((v) => v.trim())
-        .filter(Boolean)
+        .map((line) => {
+          if (!line) return null
+          if (!line.trim().startsWith('data:')) {
+            console.warn(`Skipping message from SSE endpoint: ${line}`)
+            return null
+          }
+          const text = line.slice(5).trim()
```

Comment:
nit: I know this is just being refactored but maybe we could have a comment along the lines of "5 is the length of 'data:'?